Mixpanel/MPNotificationViewController.m

```diff
@@ -94,42 +101,61 @@ - (void)viewDidLoad
+            self.bodyLabel.text = notification.body;
+            self.titleLabel.textColor = UIColorFromRGB(notification.titleColor);
+            self.bodyLabel.textColor = UIColorFromRGB(notification.bodyColor);
+        } else {
+            [[NSLayoutConstraint constraintWithItem:_titleLabel attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:nil attribute:NSLayoutAttributeNotAnAttribute multiplier:1 constant:0] setActive:YES];
+            [[NSLayoutConstraint constraintWithItem:_bodyLabel attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:nil attribute:NSLayoutAttributeNotAnAttribute multiplier:1 constant:0] setActive:YES];
         }
-        
-        if ([self.notification.style isEqualToString:@"light"]) {
-            self.viewMask.backgroundColor = [UIColor whiteColor];
-            self.titleView.textColor = [UIColor colorWithRed:92/255.0 green:101/255.0 blue:120/255.0 alpha:1];
-            self.bodyView.textColor = [UIColor colorWithRed:123/255.0 green:146/255.0 blue:163/255.0 alpha:1];
-            self.okayButton.isLight = YES;
-            [self.okayButton setTitleColor:[UIColor colorWithRed:123/255.0 green:146/255.0 blue:163/255.0 alpha:1] forState:UIControlStateNormal];
-            self.okayButton.layer.borderColor = [UIColor colorWithRed:218/255.0 green:223/255.0 blue:232/255.0 alpha:1].CGColor;
-            UIImage *origImage = [self.closeButton imageForState:UIControlStateNormal];
-            UIImage *tintedImage = [origImage imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
-            [self.closeButton setImage:tintedImage forState:UIControlStateNormal];
-            self.closeButton.tintColor = [UIColor colorWithRed:217/255.0 green:217/255.0 blue:217/255.0 alpha:1];
+
+        UIImage *originalImage = self.closeButton.imageView.image;
+        UIImage *tintedImage = [originalImage imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+        [self.closeButton setImage:tintedImage forState:UIControlStateNormal];
+        self.closeButton.tintColor = UIColorFromRGB(notification.closeButtonColor);
+
+        if (!notification.shouldFadeImage) {
+            self.bottomImageSpacing.constant = 30;
+            self.fadingView.layer.mask = nil;
+        }
+
+        [self setUpButtonView:self.firstButton withData:notification.buttons[0] forIndex:0];
+
+        if (notification.buttons.count == 2) {
+            [self setUpButtonView:self.secondButton withData:notification.buttons[1] forIndex:1];
+        } else {
+            [[NSLayoutConstraint constraintWithItem:_secondButtonContainer attribute:NSLayoutAttributeWidth relatedBy:NSLayoutRelationEqual toItem:nil attribute:NSLayoutAttributeNotAnAttribute multiplier:1 constant:0] setActive:YES];
         }
+        
+        self.viewMask.backgroundColor = UIColorFromRGB(notification.backgroundColor);
+        self.viewMask.clipsToBounds = YES;
+        self.viewMask.layer.cornerRadius = 6.f;
```

Comment:
I personally prefer to specify what type of number a variable is expecting so it's easier for someone else to modify it without having to check for `cornerRadius` type.